### ISSUE TITLE
Crashing in 4.4.4 Arch Linux

### ISSUE THREAD

**Issue by sheeb1** [musescore/MuseScore#25918]
### Issue type

Crash or freeze

### Description with steps to reproduce

1. Open musescore
2. Create new score
3. Start writing (recreation of what i was writing attached)
4. It crashes
[bug.pdf](https://github.com/user-attachments/files/18241333/bug.pdf)

### Supporting files, videos and screenshots

![image](https://github.com/user-attachments/assets/b7c702f7-ba11-4772-b755-0b99cab25461)
^While writing, crashed ~15 seconds later

### What is the latest version of MuseScore Studio where this issue is present?

4.4.4

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Arch Linux

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by MarcSabatella** [comment]
It's impossible to say from just a picture how you went about adding that music.  Please ZIP and attach the score at the state just before the step that causes the crash, then give the precise instructions to recreate the crash from there.

It kind of sounds like you are saying the crash happens not in response to a specific action.  In which case it could be the autosave kicking in and the crash happens there.  Again, try saving your score explicitly.

Also, be sure to be using the official supported build of MuseScore Studio - the AppImage you download directly from musescore.org or GitHub.  Unofficial / unsupported third party builds often have issues that lead to crashes and other problems that don't happen with the trusted builds.

**Comment by sheeb1** [comment]
[testscore.zip](https://github.com/user-attachments/files/18241868/testscore.zip)
I'm using the arch linux package for musescore which says its upstream URL is musescore.org, would this also be official?
https://archlinux.org/packages/extra/x86_64/musescore/

**Comment by Jojo-Schmitz** [comment]
No, on the AppImage from https://musescore.org/en/download/musescore-x86_64.AppImage or those from [GitHub](https://github.com/musescore/MuseScore/releases) are official and supported here

**Comment by seascallion** [comment]
I am seeing this issue too, on v4.4.4 on Arch Linux. Here's how I repro it on that test file:

1. Select the quarter note in Contrabasses measure 5
2. Set it to an eighth note (shortcut: `4`)
3. Set it back to a quarter note (shortcut: `5`)

This reliably causes a failure with these logs:

```
14:47:38.670 | INFO  | main_thread     | ActionsDispatcher::dispatch | try call action: pad-note-4
/usr/include/c++/14.2.1/bits/stl_vector.h:1237: std::vector<_Tp, _Alloc>::reference std::vector<_Tp, _Alloc>::back() [with _Tp = mu::engraving::MeasureBase*; _Alloc = std::allocator<mu::engraving::MeasureBase*>; reference = mu::engraving::MeasureBase*&]: Assertion '!this->empty()' failed.
zsh: IOT instruction (core dumped)  mscore -d test.mscz
```

The crash seems to be happening most often when overwriting eighth rests.

**Comment by seascallion** [comment]
Ok, I tried the same on the official download build (AppImage), and it didn't crash. I'm curious what the build process that creates the official Arch version, and how it's different from that one. Normally my workflow involves using pacman and yay to install and upgrade all packages, but might need to maintain this program separately for this reason.

**Comment by MarcSabatella** [comment]
Presumably they have turned on debug mode or otherwise enabled assertion failures.  That is but one of very many things third parties tend to get wrong when trying to build MuseScore.  So even if they managed to fix this particular problem, it’s still not going to be a good idea to use anything but the official build, which is also trivially simple to update (run from command with “update” option).

**Comment by rolodoom** [comment]
Maybe the installation instructions for the Appimage need to be more accessible on the official site, like just after you click the download button? I learn to Install and update the official AppImage, but don't even remember how I got there, but I can tell you it was no an easy journey. I tried arch build, flatpak, snaps, etc, and then , when I learn how to install from the AppImage there was no turning back. Cheers.

https://musescore.org/en/handbook/4/download-and-installation

**Comment by cbjeukendrup** [comment]
(Plus that this problem seems to be covered by https://github.com/musescore/MuseScore/issues/24648 and https://github.com/musescore/MuseScore/issues/24705)
